partiql-ast/Cargo.toml

```diff
@@ -20,7 +20,7 @@ path = "src/lib.rs"
 bench = false
 
 [dependencies]
-indexmap = "1.9"
+indexmap = { version = "1.9", default-features = false, features = ["serde"] }
```

Comment:
You'll want to remove the unconditional `features = ["serde"]`, since you turn it on below conditionally via `serde = [... "indexmap/serde"]`

Reply:
Whoops, let fix that.